install.sh

```diff
@@ -38,14 +38,15 @@ BASE_TEMPLATE_CONTENT='#!/bin/sh
 #   0 when successfully finished, 1 otherwise
 #####################################################
 process_git_hook() {
+    set_main_variables
+    register_for_autoupdate_if_needed
```

Comment:
We always register (meaning this repo might be prone to updates, which is good). 
No matter if the hooks are disabled.